src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -264,7 +266,7 @@ tests:
       response:
           value: 0x0001
           constraints:
-              type: groupid
+              type: int16u
```

Comment:
Again, why this change?  Per spec this should in fact be a group-id.  If our YAML processor is not handling that right, where is the issue with links to it from the YAML tracking that?